src/Sulu/Bundle/HttpCacheBundle/DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,14 @@ public function getConfigTreeBuilder()
                                     ->defaultNull()
                                     ->info('Default host name and optional path for path based invalidation.')
                                 ->end()
+                                ->enumNode('tag_mode')
+                                    ->info('If you can enable the xkey module in Varnish, use the purgekeys mode for more efficient tag handling')
```

Comment:
```suggestion
                                    ->info('Use the purgekeys mode for more efficient tag handling, if your Varnish server supports the xkey module')
```

Reply:
copied from the library :) but will change it